doc/faq.rst

```diff
@@ -5,9 +5,9 @@ What is a Binder?
 -----------------
 
 A Binder is a Git repository that has been outfitted with the appropriate
-`build files <http://repo2docker.readthedocs.io/en/latest/samples.html>`_ so
```

Comment:
I think https://repo2docker.readthedocs.io/en/latest/config_files.html would be the new link for this broken link.

Reply:
Why link to repo2docker docs instead of https://mybinder.readthedocs.io/en/latest/config_files.html

Reply:
Because the mybinder.org page is sources from the repo2docker docs (there is some magic in the build process) and I think we should start linking out to the original docs. The goal is the make docs.mybinder.org a page that shows docs that is relevant to the specific BinderHub instance. So that everyone who deploys a BinderHub can also deploy this doc page. I think they should all link to the repo2docker docs which is the one that is up-to-date.